app/styles/ui/_merge-status.scss

```
@@ -53,6 +52,10 @@
       color: var(--text-color);
     }
   }
+
+  .merge-message {
+    padding-bottom: 10px;
```

We can use a spacing variable here:
```suggestion
    padding-bottom: var(--spacing);
```